packages/pipeline-editor/src/CustomFormControls/StringArrayControl.tsx

```diff
@@ -316,6 +316,17 @@ function StringArrayControl({ placeholder, format }: Props) {
     [actionHandler, handleAction, items]
   );
 
+  const handleNewValues = useCallback(async () => {
+    const updatedProperties = await actionHandler?.(
+      "update_properties",
```

Comment:
can we change this to `request_properties_update` or `refresh_properties`? I keep thinking `update_properties` is the builtin `UPDATE_PROPERTY` action

Reply:
done, will be in next commit